requirements/edx/github.in

```diff
@@ -92,5 +92,4 @@
 # Legacy URL dependencies. To be removed. Don't add new URLs here!
 ##############################################################################
 
-# https://github.com/openedx/MongoDBProxy/issues/18
```

Comment:
🎉 

You can remove the whole "Legacy URLs" heading too--MongoDbProxy is the last one! The edx-drf-extension pin should have been put under "release candidates testing".

Reply:
Removed the `Legacy URLs` section. `drf-extension` pin has already been removed.